### PR TITLE
added minwidth attribute to data-col

### PR DESCRIPTION
**Pull request by 50Hannah000**

### CODE REVIEW

File: tests/integration/components/data-col-test.js [hunk 1]
```diff
@@ -81,6 +82,7 @@ test('styles the cell with given styling', function(assert) {
 
   let em    = parseInt($cell.css('font-size').replace('px', ''));
   let width = 20 * em;
+  let minwidth = 15 * em;
```
Comment (michaelw8) [inline tests/integration/components/data-col-test.js]: - minWidth
- align spaces

File: tests/integration/components/data-col-test.js [hunk 2]
```diff
@@ -93,6 +95,7 @@ test('styles the cell with given styling', function(assert) {
   assert.equal($cell.css('border-width'),     '4px',            'borderWidth');
   assert.equal($cell.css('border-color'),     'rgb(0, 0, 255)', 'borderColor');
   assert.equal($cell.css('border-style'),     'dotted',         'borderStyle');
+  assert.equal($cell.css('min-width'),        `${minwidth}px`,    'minWidth');
```
Comment (cbroeren) [inline tests/integration/components/data-col-test.js]: Alignment of last

File: addon/components/data-col.js [hunk 3]
```diff
@@ -26,7 +26,8 @@ const STYLING_PROPERTIES = {
   fontColor:       { key: 'color' },
   borderWidth:     { suffix: 'px' },
   borderColor:     { },
-  borderStyle:     { }
+  borderStyle:     { },
+  minWidth:        { suffix: 'em' }
```
Comment (cbroeren) [inline addon/components/data-col.js]: I prefer placing the `minWidth` after `width`

**Comment by cbroeren** [comment]
Version 0.0.28

**Pull request merged**